lib/rules/at-rule-descriptor-value-no-invalid/README.md

```diff
@@ -0,0 +1,60 @@
+# at-rule-descriptor-value-no-invalid
+
+Disallow invalid values for descriptors within at-rules.
+
+<!-- prettier-ignore -->
+```css
+@counter-style foo {
+  system: unknown;
+/**       ↑
+ * Values like this */
+}
+```
+
+This rule considers descriptors and values defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
+
+You can filter the [CSSTree Syntax Reference](https://csstree.github.io/docs/syntax/) to find out what values are valid for a descriptor of an at-rule.
+
+The [`message` secondary option](../../../docs/user-guide/configure.md#message) can accept the arguments of this rule.
+
+This rule checks descriptor values within at-rules. You can use [`declaration-property-value-no-unknown`](../declaration-property-value-no-unknown/README.md) rule to disallow unknown values for properties within declarations.
```

Comment:
```suggestion
This rule checks descriptor values within at-rules. You can use [`declaration-property-value-no-unknown`](../declaration-property-value-no-unknown/README.md) to disallow unknown values for properties within declarations, and [`at-rule-descriptor-no-unknown`](../at-rule-descriptor-no-unknown/README.md) to disallow unknown descriptors for at-rules.
```

Let's add a reference to the `at-rule-descriptor-no-unknown` rule as it touches the same area, and people will likely use them in conjunction.